### DISCUSSION TITLE
Software interrupts from timer

### DISCUSSION THREAD

**Discussion by ogogon** [earlephilhower/arduino-pico#1586] [Q&A]
Greetings!
How can you initialize one of the timers using your package and make it a software interrupt source?

**Comment by earlephilhower** [comment]
You have full access to the Pico SDK, so look at their `alarm`s section https://www.raspberrypi.com/documentation/pico-sdk/high_level.html#alarm .  There are no specific library wrappers here.  You may want to look also at some of the `Ticker` clones which have something similar.
